lib/helpers/UiBaseLocators.ts

```diff
@@ -79,6 +80,7 @@ export class UiBaseLocators {
     this.saveBtn = page.getByLabel('Save', {exact: true});
     this.submitBtn = page.getByLabel('Submit');
     this.deleteExactLabelBtn = page.getByLabel('Delete', {exact: true});
+    this.deleteExactBtn = page.getByRole('button', { name: 'Delete', exact: true })
```

Comment:
```suggestion
    this.deleteExactBtn = page.getByRole('button', {name: 'Delete', exact: true});
```
It would be better if you can use "deleteBtn" instead of defining a new element.